src/Installer.php

```diff
@@ -56,7 +56,6 @@ class Installer
 --version="..."      install a specific version
 --filename="..."     set the target filename (default: puli.phar)
 --disable-tls        disable SSL/TLS security for file downloads
---cafile="..."       set the path to a Certificate Authority (CA) certificate file for SSL/TLS verification
```

Comment:
It doesn't seems so, at least not without extending it: https://github.com/padraic/file_get_contents/blob/master/src/Humbug/FileGetContents.php#L28

But I'm not sure.